spk/tt-rss/Makefile

```diff
@@ -1,6 +1,6 @@
 SPK_NAME = tt-rss
-SPK_VERS = 20160102
-SPK_REV = 10
+SPK_VERS = 20170422
```

Comment:
the SPK_REV field actually determines whether the Synology Package manager actually thinks it is a new version or not